src/lib/components/pager/terra-pager.component.ts

```diff
@@ -14,109 +7,95 @@ import { Language } from 'angular-l10n';
 import { debounceTime } from 'rxjs/operators';
 
 @Component({
-    selector:    'terra-pager',
-    styleUrls:   ['./terra-pager.component.scss'],
+    selector: 'terra-pager',
+    styleUrls: ['./terra-pager.component.scss'],
     templateUrl: './terra-pager.component.html'
 })
-export class TerraPagerComponent implements OnInit, OnDestroy
-{
+export class TerraPagerComponent implements OnInit, OnDestroy {
     @Input()
-    public inputPagingData:TerraPagerInterface<any>;
+    public inputPagingData: TerraPagerInterface<any>;
 
     @Input()
-    public inputDefaultPagingSize:number;
+    public inputDefaultPagingSize: number;
 
     @Input()
-    public inputPagingSize:Array<TerraSelectBoxValueInterface>;
+    public inputPagingSize: Array<TerraSelectBoxValueInterface>;
 
     @Input()
-    public inputRequestPending:boolean;
+    public inputRequestPending: boolean;
 
     @Output()
-    public outputDoPaging:EventEmitter<TerraPagerInterface<any>> = new EventEmitter<TerraPagerInterface<any>>();
+    public outputDoPaging: EventEmitter<TerraPagerInterface<any>> = new EventEmitter<TerraPagerInterface<any>>();
 
     @Language()
-    public _lang:string;
+    public _lang: string;
 
-    private _pagingClicks:Subject<TerraPagerInterface<any>> = new Subject<TerraPagerInterface<any>>();
+    private _pagingClicks: Subject<TerraPagerInterface<any>> = new Subject<TerraPagerInterface<any>>();
 
-    public ngOnInit():void
-    {
-        this._pagingClicks.pipe(debounceTime(400)).subscribe((e:TerraPagerInterface<any>) => this.outputDoPaging.emit(e));
+    public ngOnInit(): void {
+        this._pagingClicks
+            .pipe(debounceTime(400))
+            .subscribe((e: TerraPagerInterface<any>) => this.outputDoPaging.emit(e));
 
-        if(!this.inputDefaultPagingSize)
-        {
+        if (!this.inputDefaultPagingSize) {
             this.inputDefaultPagingSize = 25;
         }
 
-        if(!this.inputPagingSize)
-        {
+        if (!this.inputPagingSize) {
             this.inputPagingSize = terraPagerDefaultPagingSizes;
         }
 
-        if(!this.inputPagingData)
-        {
+        if (!this.inputPagingData) {
             this.inputPagingData = {
-                pagingUnit:     '',
-                totalsCount:    0,
-                page:           1,
-                itemsPerPage:   0,
+                pagingUnit: '',
```

Comment:
wird das hier dann der Standard-Formattierung? Ich finde es halt vorher viel ordentlicher und lesbarer. Ein Tabulaturdamit alle Werte untereinander aufgelistet sind ist doch übersichtlicher

Reply:
Ja, wird es :) Die Übersichtlichkeit geht auch bei dem Tabulator verloren, wenn die Keys sehr unterschiedlich lang sind, weil dann einige Werte "in der Luft schweben" und es nicht so leicht zu erfassen ist wozu der Wert genau gehört.

Reply:
ja, ist halt alles relativ :) . Prinzipiell habe nichts dagegen das so zu machen, aber meine Vorliebe liegt bei der alten Formatierung. 👍